### PR TITLE
Move to common

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Depends on #41

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
Review suggestion: Just look at the last commit. The 2 others are from #41.

File: mozapkpublisher/get_apk.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 import shutil
 import logging
 
-from mozapkpublisher.base import Base, ArgumentParser
-from mozapkpublisher.utils import load_json_url, download_file, file_sha512sum
-from mozapkpublisher.exceptions import CheckSumMismatch
+from mozapkpublisher.common.base import Base, ArgumentParser
+from mozapkpublisher.common.utils import load_json_url, download_file, file_sha512sum
+from mozapkpublisher.common.exceptions import CheckSumMismatch
```
Comment (MihaiTabara) [inline mozapkpublisher/get_apk.py]: Super-super nit: I'm not sure if PEP8 references the imported submodules as well in alphabetical order but might be worth importing the exception before utils.

File: mozapkpublisher/test/common/test_googleplay.py [hunk 2]
```diff
@@ -8,8 +8,8 @@
 except ImportError:
     from mock import MagicMock
 
-from mozapkpublisher.googleplay import add_general_google_play_arguments, EditService, is_package_name_nightly
-from mozapkpublisher.exceptions import NoTransactionError, WrongArgumentGiven
+from mozapkpublisher.common.googleplay import add_general_google_play_arguments, EditService, is_package_name_nightly
+from mozapkpublisher.common.exceptions import NoTransactionError, WrongArgumentGiven
```
Comment (MihaiTabara) [inline mozapkpublisher/test/common/test_googleplay.py]: Super-super-nit: swap the lines to be alphabetically consistent.

**Comment by JohanLorenzo** [comment]
Good point for the alphabetical order. Will do!

**Pull request merged**